pkgs/applications/misc/ding/default.nix

```diff
@@ -0,0 +1,52 @@
+{
+aspell,
+gnugrep,
```

Comment:
AFAIK, it should be added into `buildInputs` and verified that ding picks up full paths.